src/utils/daos_hdlr.c

```diff
@@ -2810,13 +2539,15 @@ dm_connect(struct cmd_args_s *ap,
 	int			rc = 0;
 	struct duns_attr_t	dattr = {0};
 	daos_prop_t		*props = NULL;
-	dfs_attr_t		attr = {0};
 	int			size = 2;
 	uint32_t		dpe_types[size];
-	uint64_t		dpe_vals[size];
+	uint64_t		*dpe_vals = NULL;
```

Comment:
Why make dpe_types static and dpe_vals dynamic since they are both [size]?

Reply:
@daltonbohning I think the reason I did this is because dpe_types is not modified in dm_get_cont_prop function (the values are set before), but the dpe_vals are set within the function.

Reply:
@daltonbohning I admit I cannot remember exact reason here.

Reply:
@daltonbohning I do remember dpe_vals needs to modify array values in dm_cont_get_prop function whereas dpe_types does not.

Reply:
@daltonbohning yeah it used to be static array. I will change it back to make less confusing.